string poseName
string chainName
string[] jointNames
---
measurementData measurement
